package.json

```diff
@@ -88,6 +89,8 @@
     "@nuxtjs/eslint-config-typescript": "latest",
     "@types/mdurl": "^1.0.2",
     "@types/ws": "^8.5.5",
+    "@vueuse/core": "^10.2.1",
+    "@vueuse/nuxt": "^10.2.1",
```

Comment:
I think we should move this to `dependencies` since it is registered via `module.ts`

Reply:
Yes, it could be better.

Reply:
Done